### PR TITLE
Richer pwa install article

### PR DESCRIPTION
**Pull request by mustafa-x**
Adding new article regarding Richer Installs UI, explaining how to implement etc.

### CODE REVIEW

File: site/en/blog/richer-pwa-installation/index.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+## Backwards compatibility {: #compatibility }
+
+Websites that do not conform to the above will continue to receive the existing prompts. This may
+change in the future depending on uptake of the developer community and users' reaction.  
+
+## Previewing the UI {: #previewing }
+
+This UI works in Chrome Dev & Chrome Canary on Android, with the flag enabled;
+
+`mobile-pwa-install-use-bottom-sheet`
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:75]: This needs to include the flags path and the flag should have a # sign in it. Here's a common boilerplate: https://github.com/GoogleChrome/web.dev/blob/main/src/site/content/en/blog/multi-screen-window-placement/index.md

File: site/en/blog/richer-pwa-installation/index.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+`mobile-pwa-install-use-bottom-sheet` 
+
+This feature is enabled on [squoosh.app](https://squoosh.app) and can be previewed there. As an
+aside, [#screenshots](https://web.dev/add-manifest/#screenshots) and 
+[#description](https://web.dev/add-manifest/#description) fields are already documented; 
+you can give it a try behind the flag. 
+
+## Implementation {: #implementation }
+
+In order to enable Richer installs on your site you need to add at least one screenshot to your
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:84]:
```suggestion
To enable Richer installs on your site you need to add at least one screenshot to your 
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+In order to enable Richer installs on your site you need to add at least one screenshot to your 
+manifest file. Descriptions are not required but are recommended. 
+
+Origins that are too long to fit the UI are truncated, this is also known as eliding and is used
+as a [security measure to protect users](https://chromium.googlesource.com/chromium/src/+/master/docs/security/url_display_guidelines/url_display_guidelines.md#eliding-urls). 
+
+Let's take a look at how we can trigger Richer Install UI when installing a PWA. In this example
+we are looking at squoosh.app implementation. Let's assume that you have your name and origin
+defined.
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:92]: Please rewrite without 'we' per the [handbook](https://web.dev/handbook/grammar/#instructions).

File: site/en/blog/richer-pwa-installation/index.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    }],
+"start_url": "/?start_url",
+"scope": "/",
+"display": "standalone",
+"background_color": "#fff",
+"theme_color": "#fff"
+}
+ ```
+
+Now let's add the screenshots to the end of the manifest.json file;
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:112]:
```suggestion
Now add the screenshots to the end of the manifest.json file; 
```

File: site/en/blog/richer-pwa-installation/index.md (hunk 3 above)
Comment (jpmedley) [reply site/en/blog/richer-pwa-installation/index.md:92]: please check this throughout.

File: site/en/blog/richer-pwa-installation/index.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+"background_color": "#fff",
+"theme_color": "#fff"
+}
+ ```
+
+Now let's add the screenshots to the end of the manifest.json file; 
+
+## Screenshots {: #screenshots }
+
+To add screenshots we need to determine the size, source and type, following this
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:116]:
```suggestion
To add screenshots you need to determine the size, source and type, following this 
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+## Screenshots {: #screenshots }
+
+To add screenshots we need to determine the size, source and type, following this 
+[criteria](https://web.dev/add-manifest/#screenshots). 
+
+Width and height must be at least 320px and at most 3840px.
+The maximum dimension can't be more than 2.3 times as long as the minimum dimension.
+Screenshots must have the same aspect ratio.
+Only JPEG and PNG image formats are supported.
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:122]:
```suggestion
* Width and height must be at least 320px and at most 3840px.
* The maximum dimension can't be more than 2.3 times as long as the minimum dimension.
* Screenshots must have the same aspect ratio.
* Only JPEG and PNG image formats are supported.
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+Currently animated gifs are not supported. We need to include the size and type of the image so it
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:124]:
```suggestion
Currently animated gifs are not supported. Also, you need to include the size and type of the image so it 
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+alt="A single screenshot added.", width="342", height="684" %}
+{% Aside %}
+A single screenshot added.
+{% endAside %}
+{% endColumn %}
+{% endColumns %}
+
+## Description {: #description }
+
+Descriptions are not compulsory, so there is no minimum requirement. There however is a max which
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:150]:
```suggestion
Descriptions are not compulsory, so there is no minimum requirement. However, there is a maximum that 
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+{% endAside %}
+{% endColumn %}
+{% endColumns %}
+
+## Description {: #description }
+
+Descriptions are not compulsory, so there is no minimum requirement. There however is a max which 
+kicks in after 7 lines of text (roughly 324 characters). After this an ellipse will appear and 
+truncate longer descriptions 
+([see example](https://glitch.com/edit/#!/richerinstall-longer-description)).
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:153]:
```suggestion
([for example](https://glitch.com/edit/#!/richerinstall-longer-description)). 
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+{% endColumns %}
+
+## Description {: #description }
+
+Descriptions are not compulsory, so there is no minimum requirement. There however is a max which 
+kicks in after 7 lines of text (roughly 324 characters). After this an ellipse will appear and 
+truncate longer descriptions 
+([see example](https://glitch.com/edit/#!/richerinstall-longer-description)). 
+
+In the future we will consider adding other data such as categories and app rating, but this will
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:155]: This 'we' is fine.

File: site/en/blog/richer-pwa-installation/index.md [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+## Description {: #description }
+
+Descriptions are not compulsory, so there is no minimum requirement. There however is a max which 
+kicks in after 7 lines of text (roughly 324 characters). After this an ellipse will appear and 
+truncate longer descriptions 
+([see example](https://glitch.com/edit/#!/richerinstall-longer-description)). 
+
+In the future we will consider adding other data such as categories and app rating, but this will 
+be based on feedback from developers and users. 
+[See example code here](https://glitch.com/edit/#!/richerinstall-description?path=manifest.json%3A13%3A29).
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:157]:
```suggestion
[See the example code here](https://glitch.com/edit/#!/richerinstall-description?path=manifest.json%3A13%3A29).
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+
+Example;
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:159]:
```suggestion
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 13]
```diff
@@ -0,0 +1,188 @@
...
+([see example](https://glitch.com/edit/#!/richerinstall-longer-description)). 
+
+In the future we will consider adding other data such as categories and app rating, but this will 
+be based on feedback from developers and users. 
+[See example code here](https://glitch.com/edit/#!/richerinstall-description?path=manifest.json%3A13%3A29).
+
+Example; 
+
+ ```javascript
+"description": "Compress and compare images with different codecs,
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:162]:
```suggestion
"description": "Compress and compare images with different codecs
```

File: site/en/blog/richer-pwa-installation/index.md [hunk 14]
```diff
@@ -0,0 +1,188 @@
...
+{% Column %}
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/oOj7Ls7cQ8E274faxfOz.jpg", 
+alt="Description added", width="342", height="684" %}
+{% Aside %}
+Description added.
+{% endAside %}
+{% endColumn %}
+{% Column %}
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/Dpzs03K6QmBkZaefX2nU.jpg", 
+alt="Longer description added that has been truncated.", width="342", height="684" %}
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:176]:
```suggestion
alt="A longer description that has been truncated.", width="342", height="684" %}
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/richer-pwa-installation/index.md [hunk 15]
```diff
@@ -0,0 +1,188 @@
...
+information. If they do continue to install, the process is over too quickly without giving
+context to the user. This goes against their expectations of installing apps and can leave them
+somewhat confused with what has happened.  
+
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/VwB1V3K61vQMs1htKJpY.png", 
+alt="An example of PWA install UI.", width="360", height="632" %}
+
+To enable and empower developers to provide installed experiences on--par with native experiences
+Chrome is introducing a new install surface, Richer Install, that will allow developers to add a
+description and screenshots to their manifest file and have it appear in a bottomsheet dialog
```
[the remark this answers is not in the document]
Comment (mustafa-x) [reply site/en/blog/richer-pwa-installation/index.md:42]: The image appears below it, with the example. Shall I add another image as well with just a bottomsheet?

File: site/en/blog/richer-pwa-installation/index.md [hunk 16]
```diff
@@ -0,0 +1,188 @@
...
+within Chrome. 
+
+By doing this we are giving developers the opportunity to create a more enticing install process
+that better aligns to user expectations and better context that mimics their existing mental model
```
[the remark this answers is not in the document]
Comment (mustafa-x) [reply site/en/blog/richer-pwa-installation/index.md:46]: removed

File: site/en/blog/richer-pwa-installation/index.md [hunk 17]
```diff
@@ -0,0 +1,188 @@
...
+alt="Richer Install UI Collapsed", width="342", height="722" %}
+{% Aside %}
+Richer Install UI Collapsed
+{% endAside %}
+{% endColumn %}
+{% endColumns %}
+
+## Backwards compatibility {: #compatibility }
+
+Websites that do not conform to the above will continue to receive the existing prompts. This may
```
[the remark this answers is not in the document]
Comment (mustafa-x) [reply site/en/blog/richer-pwa-installation/index.md:68]: I updated it to this
'Websites that do not include at least one screenshot to their manifest file will continue to receive the existing prompts.'

File: site/en/blog/richer-pwa-installation/index.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/blog/richer-pwa-installation/index.md:42]: Unless I'm missing something, I can't find that it's labeled as such. That means I have to look it up. Please make this clear.

File: site/en/blog/richer-pwa-installation/index.md [hunk 18]
```diff
@@ -0,0 +1,185 @@
...
+
+## Backwards compatibility {: #compatibility }
+
+Websites that do not include at least one screenshot to their manifest file will continue to receive 
+the existing prompts. This may change in the future depending on uptake of the developer community 
+and users' reaction.  
+
+## Previewing the UI {: #previewing }
+
+This UI works in Chrome Dev & Chrome Canary on Android, with the flag `#mobile-pwa-install-use-bottom-sheet` enabled in `chrome://flags`.
```
Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:74]: This is what I was commenting on at the top. Please use specific version numbers.

Comment (jpmedley) [inline site/en/blog/richer-pwa-installation/index.md:74]:
```suggestion
This UI works in Chrome Dev and Chrome Canary on Android, with the flag `#mobile-pwa-install-use-bottom-sheet` enabled in `chrome://flags`.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/richer-pwa-installation/index.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (mustafa-x) [reply site/en/blog/richer-pwa-installation/index.md:42]: added an image with an example of bottomsheet UI to be clearer.

File: site/en/blog/richer-pwa-installation/index.md [hunk 19]
```diff
@@ -0,0 +1,188 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Richer PWA installation UI
```
[the remark this answers is not in the document]
Comment (mustafa-x) [reply site/en/blog/richer-pwa-installation/index.md:3]: Updated with the milestone

**Comment by jpmedley** [comment]
I'll handle it. 

Sorry about that.

**Pull request merged**